core-bundle/tests/Doctrine/Backup/DumperTest.php

```diff
@@ -68,7 +68,7 @@ public function testUnsuccessfulDump(): void
     public function successfulDumpProvider(): \Generator
     {
         yield 'Empty table without data' => [
-            [new Table('tl_page', [new Column('foobar', Type::getType(Types::STRING))])],
+            [new Table('tl_page', [new Column('foobar', Type::getType(Types::STRING), ['length' => 255])])],
```

Comment:
```suggestion
            [new Table('tl_page', [new Column('foobar', Type::getType(Types::STRING), ['length' => 255])], [], [], [], ['charset' => 'utf8', 'collation' => 'utf8_unicode_ci', 'engine' => 'InnoDB'])],
```

I think we should test if the options still get dumpted.